docs/source/user-guide/vision/quickstarts/plot_simple_classification_tutorial.py

```diff
@@ -7,12 +7,13 @@
 
 Deepchecks Vision is built to validate your data and model, however complex your model and data may be. That
 being said, sometime there is no need to write a full-blown
-:doc:`ClassificationData </user-guide/vision/auto_tutorials/plot_classification_tutorial>` or
-:doc:`DetectionData </user-guide/vision/auto_tutorials/plot_detection_tutorial>`. In the case of a simple classification
-task, there is quite a few checks that can be run writing only a few lines of code. In this tutorial, we will show you
+:doc:`classification task </user-guide/vision/auto_tutorials/plot_classification_tutorial>`,
+:doc:`object detection task </user-guide/vision/auto_tutorials/plot_detection_tutorial>` or
+:doc:`semantic segmentation task </user-guide/vision/auto_tutorials/plot_segmentation_tutorial>`. In the case of a simple classification
+task, there are quite a few checks that can be run writing only a few lines of code. In this tutorial, we will show you
 how to run all checks that do not require a model on a simple classification task.
 
-This is ideal, for example, when receiving a new dataset for a classification task. Running these checks on the dataset
+This is ideal, for example, when receiving a new dataset for a any given task. Running these checks on the dataset
```

Comment:
hope that was true but currently it is not... (only good for classification)

Reply:
Oops